build.js

```diff
@@ -62,7 +62,8 @@ const cdnScriptsOptional = {
   '{{version}}/modules/coloraxis.js': 1,
   '{{version}}/modules/venn.js': 1,
   "{{version}}/modules/dumbbell.js": 1,
-  '{{version}}/modules/lollipop.js': 1
+  '{{version}}/modules/lollipop.js': 1,
+  '{{version}}/modules/no-data-to-display.js' : 1
```

Comment:
```suggestion
  '{{version}}/modules/lollipop.js': 1
```

I believe a module shouldn't be added both to the Optional list as well as the Common list. It should either be optional or always included. Not sure, but I think the module is now included/downloaded twice.

I have created a PR to fix this (which also includes another module I needed): https://github.com/highcharts/node-export-server/pull/296

Reply:
sure, you can merge that one.
So we can close this one.

Reply:
Thanks for the quick response. Can you close this PR? I cannot close PRs of another author.